---
title: "' ' は ' <name1> ' から継承されるため、' Catch ' ブロックに到達しませんでした。 <name2>"
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: 06f3b3cb69d2df5d123879fa0df3dffcdc661678
ms.sourcegitcommit: bf5c5850654187705bc94cc40ebfb62fe346ab02
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 09/23/2020
ms.locfileid: "91084581"
---
# <a name="catch-block-never-reached-because-name1-inherits-from-name2"></a><span data-ttu-id="639f3-102">' ' は ' \<name1> ' から継承されるため、' Catch ' ブロックに到達しませんでした。 \<name2></span><span class="sxs-lookup"><span data-stu-id="639f3-102">'Catch' block never reached, because '\<name1>' inherits from '\<name2>'</span></span>

<span data-ttu-id="639f3-103">コード内の `Catch` ブロックに到達することはありません。</span><span class="sxs-lookup"><span data-stu-id="639f3-103">A `Catch` block within the code is never reached.</span></span>  
  
 <span data-ttu-id="639f3-104">既定では、このメッセージは警告です。</span><span class="sxs-lookup"><span data-stu-id="639f3-104">By default, this message is a warning.</span></span> <span data-ttu-id="639f3-105">警告を非表示にする方法や、警告をエラーとして扱う方法の詳細については、「 [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic)」を参照してください。</span><span class="sxs-lookup"><span data-stu-id="639f3-105">For more information on hiding warnings or treating warnings as errors, see [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span></span>  
  
 <span data-ttu-id="639f3-106">**エラー ID:** BC42029</span><span class="sxs-lookup"><span data-stu-id="639f3-106">**Error ID:** BC42029</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="639f3-107">このエラーを解決するには</span><span class="sxs-lookup"><span data-stu-id="639f3-107">To correct this error</span></span>  
  
- <span data-ttu-id="639f3-108">余分な `Catch` ブロックを削除します。</span><span class="sxs-lookup"><span data-stu-id="639f3-108">Remove the extraneous `Catch` block.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="639f3-109">こちらもご覧ください</span><span class="sxs-lookup"><span data-stu-id="639f3-109">See also</span></span>

- [<span data-ttu-id="639f3-110">Try ブロックと Catch ブロックを使用して例外をキャッチする方法</span><span class="sxs-lookup"><span data-stu-id="639f3-110">How to use the try/catch block to catch exceptions</span></span>](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [<span data-ttu-id="639f3-111">Try...Catch...Finally ステートメント</span><span class="sxs-lookup"><span data-stu-id="639f3-111">Try...Catch...Finally Statement</span></span>](../language-reference/statements/try-catch-finally-statement.md)
